src/components/ListItem/ListItem.stories.tsx

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable max-lines */
+import React, { useState } from 'react';
+import { View, FlatList, Text } from 'react-native';
+import { Card } from '../Card';
+import { Divider } from '../Divider';
+import { ListItem, ListItemFeedback } from './ListItem';
+import { Checkbox } from '../Checkbox';
+import { TextWithTheme } from '../../common/HelperComponents/ThemeHelper.styles';
+import { data } from './stories.data';
+
+export default {
+  component: ListItem,
+  parameters: {
+    componentSubtitle: 'List items are rows information that can have user interactions or not',
```

Comment:
rows of information? desculpa a chatice, documentaçao né ><

Reply:
não entendi a duvida

Reply:
mas se a duvida é da escolha das palavras, eu tirei daqui: https://developer.apple.com/design/human-interface-guidelines/watchos/elements/lists-and-tables/

Reply:
é que faltou o OF

Reply:
AAAAAAH